CMakeLists.txt

```diff
@@ -11,9 +11,15 @@ find_package(CURL           REQUIRED)  # FIXME depends
 find_package(enet           REQUIRED)
 find_package(moonlight      REQUIRED)
 find_package(OpenSSL        REQUIRED)
-find_package(pugixml        REQUIRED)
 find_package(ZLIB           REQUIRED)
 
+find_path(TINYXML2_INCLUDE_DIRS NAMES tinyxml2.h)
+find_library(TINYXML2_LIBRARIES NAMES libtinyxml2.a)
```

Comment:
Why only search for the static variant?
This will also fail on windows as there the file extension is `.lib`